prismic-model/js/exhibition-formats.js

```diff
@@ -0,0 +1,10 @@
+import structuredText from './parts/structured-text';
+
+const ExhibitionFormats = {
+  'Exhibition format': {
+    title: structuredText('Title', 'single', ['h1']),
```

Comment:
You could just use the title type:
https://github.com/wellcometrust/wellcomecollection.org/blob/master/prismic-model/js/parts/title.js